features/shop/cart_with_different_taxes_for _different_zones.feature

```diff
@@ -0,0 +1,48 @@
+@ui-cart
+Feature: Cart with different tax rates for different time zones
+    In order to order products with different tax rate for different zones
+    As a Customer
+    I want to be aware of taxes applied on my order
+
+    Background:
+        Given that store is operating on the France channel
+        And default currency is "EUR"
+        And tax rate "EU VAT" with 23% rate belongs to "Taxable Goods" category for "EU" zone
```

Comment:
And store has "EU VAT" tax rate of 23% for "Taxable Goods" in "EU" zone
I'd use a different example here, Taxable Goods is too general, let's say it is food category or something like WDYT?